waypoints:
  waypoints_file_name: "waypoints_sim.csv" # file name of the reference waypoints file
  resolution: 0.05 # distance between interpolated waypoints

pure_pursuit_controller:
  lookahead_dist: 1.0 # static lookahead distance
  K_dd: 1 # for dynamic lookahead distance: K_dd_*velocity = lookahead dist
  length_robot: 1.12
  velocity: 0.5 # driving speed during repeat

simulation_params: 
  state_estimation_topic: "/odom"
  world_frame: "map"

real_robot_params:
  state_estimation_topic: "/rowesys/estimator/pose_fused"
  world_frame: "ENU"

#robot_footprint: [ [-1.4, 0.5], [-1.4, -0.5], [0.7, -0.5], [0.7, 0.5] ]
#robot_footprint: { {-1.4, 0.5}, {-1.4, -0.5}, {0.7, -0.5}, {0.7, 0.5} }

